abm/projects/cooperative_signaling/cs_agent/tests/test_cs_supcalc.py

```diff
@@ -12,3 +12,24 @@ def test_random_walk():
 
     assert dvel == 1.0
```

Comment:
for me random walk test fails can you please check?

Reply:
Ah it is because the env parameters, we might want to mock this so that a new local env file won't influence the test results

Reply:
If you replace the .env file (i.e. you are after running the playground tool) the tests won't pass. Or do they pass for you even with a new .env file?